cli/lib/exec/spawn.js

```diff
@@ -141,9 +141,22 @@ module.exports = {
           stdioOptions.env.DISPLAY = process.env.DISPLAY
         }
 
+        if (process.env.ELECTRON_RUN_AS_NODE === 'true') {
+          debug('passing ELECTRON_RUN_AS_NODE=true')
+          stdioOptions.env.ELECTRON_RUN_AS_NODE = 'true'
+
+          // Replace electron specific args with path to js entry point.
+          electronArgs.splice(
+            0,
+            // Electron's command line switches come before the "--" delimiter.
+            electronArgs.indexOf('--') + 1,
```

Comment:
this splice is kinda weird. looking at the code, the only electron-specific arg i see being added to the front of `electronArgs` is `--no-sandbox`.

you could:

* maintain a separate `electronArgs` and `args` arrays, and concatenate them at the end only if ELECTRON_RUN_AS_NODE is falsy
* only push `--no-sandbox` if ELECTRON_RUN_AS_NODE is falsy (on line 116)

i think either would be clearer than this

Reply:
Other electron args are passed in files other than this one (this is why --no-sandbox is added to the front of the array of args, so that it comes before the '--' that is already present)

Happy to split the args on the pre-existing '--' and do it that way though

Reply:
Done